src/nexus/backend/model_exec.h

```diff
@@ -55,6 +55,8 @@ class ModelExecutor {
 
   int NumberOfOpenRequests() const;
 
+  int64_t get_peak_memory_usage() { return model_->get_peak_bytes_in_use(); }
```

Comment:
I think it would be nice to follow the style convention of Nexus (though not explicit written anywhere, I guess the style is very close to Google. I should probably clang-format the project some time.) and use UpperCamelCase as function names.

Reply:
Change to UpperCamelCase.